src/styles/components/EventDescriptionModal.module.css

```diff
@@ -1,27 +1,43 @@
 .container {
-  width: 100svw;
-  height: 100svh;
-  position: fixed;
-  inset: 0;
-  z-index: 999;
-  background-color: rgba(10, 10, 10, 0.6);
+  display: flex;
+  gap: 1rem;
+  align-items: stretch;
+  justify-content: center;
 }
 
 .outerModalContainer {
-  display: grid;
+  display: flex;
   grid-template-columns: 50% 0 50%;
   grid-template-rows: 1fr;
   max-height: 90svh;
+  padding: 1rem;
 }
 
 .window {
-  position: absolute;
+  z-index: 99;
+  /* position: absolute;
```

Comment:
i thinkk this position absolute might be necessary, im seeing it shift the grid, which shouldn't be happening

(the three events were in a line, but clicking on description made the popup show up (good) and they moved around the events to 2 columns (not good))

<img width="1039" alt="≤ 600 PM - 700 PM" src="https://github.com/user-attachments/assets/16711a72-906c-46dc-8380-e1a2727ed4be">